project-base/app/schema.graphql

```diff
@@ -1627,8 +1627,8 @@ input ProductListInput {
 
 "One of possible types of the product list"
 enum ProductListTypeEnum {
-  COMPARISON
-  WISHLIST
+  comparison
+  wishlist
```

Comment:
Does this change require any change on the storefront?

Reply:
good catch, it actually works both ways but I did not want to change it, fixed in 3f42c88107